src/site/content/en/progressive-web-apps/web-on-android/index.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Web on Android
+authors:
+  - andreban
+date: 2020-07-30
+updated: 2020-07-30
+description: |
+  Learn about how different components can be used to render web content inside Android apps.
+tags:
+  - android
+---
+
+## Web on Android
```

Comment:
```suggestion
```

Just a drive-by redundancy that I caught while browsing this PR

![image](https://user-images.githubusercontent.com/4713486/88983893-10a3b600-d281-11ea-8c33-b239fbc737c0.png)